python/lesson3/tutorial.md

```diff
@@ -56,42 +56,40 @@ Now that we have defined a parameter, the function will raise an error if no par
 
-    >>> printed_name = print_name("Lisa")
-    >>> print_name(printed_name)
+    >>> name_statement = get_name_statement("Lisa")
+    >>> print_name(name_statement)
     'My name is My name is Lisa'
 
 ## Why use functions
 
-At first glance it may not seem obvious why it is worth expending the extra effort of defining and calling a function, rather than just writing the code independant of such things. Defining tasks as functions reduces the need to copy and paste the same code multiple times to achieve the same effect. Simply calling the function multiple times makes it much easier to not only write code, but to read it also. 
-
-Using functions also makes it a lot easier to fix and change code. If you are performing the same tasks in mutiple places and discover a bug, without functions you would need to fix the same bug multiple times. However with functions, you simply fix the bug once, and all of the subsequent function calls will now behave accordingly.
+At first glance it may not seem obvious why it is worth expending the extra effort of defining and calling a function, rather than just writing the code independent of such things. Defining tasks as functions reduces the need to copy and paste the same code multiple times to achieve the same effect. Simply calling the function multiple times makes it much easier to not only write code, but to read it also.
 
+Using functions also makes it a lot easier to fix and change code. If you are performing the same tasks in multiple places and discover a bug, without functions you would need to fix the same bug multiple times. However with functions, you simply fix the bug once, and all of the subsequent function calls will now behave accordingly.
 
 ## Further reading
 
-Python comes with a lot of built in functions that are worth learning about, as they have been created specifically to perform the most common tasks that developers require. They are listed in the [documentation](https://docs.python.org/2/library/functions.html "Python 2 built in functions documentation"). To begin with, try using a few to see what they do, and how they behave with varying parameters. Once you have grasped how a few of them work, try writing your own functions that call the built in functions within them, like we did when we used the `print` function in our own custom `print_name()` function.
+Python comes with a lot of built-in functions that are worth learning about, as they have been created specifically to perform the most common tasks that developers require. They are listed in the [documentation](https://docs.python.org/3/library/functions.html "Python 3 built in functions documentation"). To begin with, try using a few to see what they do, and how they behave with varying parameters. Once you have grasped how a few of them work, try writing your own functions that call the built in functions within them, like we did when we used the `print` function in our own custom `print_name()` function.
```

Comment:
Nice. Could also recommend this useful reference: https://learnxinyminutes.com/docs/python3/